src/Temporalio/Client/HttpConnectProxyConfig.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+
+namespace Temporalio.Client
+{
+    /// <summary>
+    /// HTTP connect proxy options for Temporal connections.
+    /// </summary>
+    public class HttpConnectProxyConfig : ICloneable
```

Comment:
```suggestion
    public class HttpConnectProxyOptions : ICloneable
```

Matches the term we've used elsewhere

Reply:
I mentioned this in the description a bit, but it seems like you all established a pattern to diffferentiate the core types from the lang types. Since the core type is already `HttpConnectProxyOptions`, I opted for the above. Plus, I wasn't sure if being consistent with the python sdk was important.

Will update.

Reply:
:+1: I think keep alive options can be used as a good template here

Reply:
Updated.